exercises/roman-numerals/roman_numerals.py

```diff
@@ -1,2 +1,2 @@
-def numeral():
+def numeral(arabic):
```

Comment:
Maybe just a `number` would suffices?
`arabic` sounds a bit strange to me

Reply:
Hi @m-a-ge 
I've changed the parameter to 'number'. Please review it again :D